packages/loader/container-loader/src/container.ts

```diff
@@ -1927,7 +1927,7 @@ export class Container
 	}
 
 	/** @returns clientSequenceNumber of last message in a batch */
-	private submitBatch(batch: IBatchMessage[]): number {
+	private submitBatch(batch: IBatchMessage[], referenceSequenceNumber?: number): number {
```

Comment:
submitSummaryMessage below also needs to be updated to have referenceSequenceNumber as a param